### ISSUE TITLE
[Pale Moon] pcmag.com not loading correctly after uBO 1.13.8

### ISSUE THREAD

**Issue by LL25255252** [gorhill/uBlock#3134]
You can reproduce it on a clean/fresh Pale Moon profile.

a page  for example :
https://www.pcmag.com/article2/0,2817,2420857,00.asp

The page loads with a a "broken" layout and with a huge image (the site's logo) at the top of the page.

This issue happens only in Pale Moon (I tested for example Waterfox 55.2)

It happens on all uBO versions after 1.13.8
Pcmag.com is the only page that causes me issues (tested on 1.14.14 and on the new 1.14.15b1)

On the uBO's latest "Read Carefully" page it says : "Those legacy extensions can cause multi-process to be disabled in your browser, and apparently when multi-process is disabled, this can cause many cases of page load failure."
https://github.com/gorhill/uBlock/wiki/Firefox-WebExtensions#future-of-ubolegacy

Can someone explain it - There is no multi-processing in Pale Moon so why does this happen ?

As of now I have now 2 options :
1. Use a workaround and disable uBO on problematic sites like pcmag.com
2. Use the uBO 1.13.8 - https://addons.mozilla.org/firefox/downloads/file/685614/ublock_origin-1.13.8-an+fx+sm+tb.xpi.

Which option do you recommend ?

Pale Moon 27.5.1 (32-bit)
Win 7 x64

Thanks in advance.

**Comment by gorhill** [comment]
> It happens on all uBO versions after 1.13.8

Incorrect, the issue on pcmag.com also affect 1.13.8. It's a filter issue, fixed in uBO filters.

Pale Moon does not support CSP Level 3, so I had to adjust a filter to work around this.

**Comment by LL25255252** [comment]
First, thank you for the quick reply.

Because you said my report was incorrect I created another new/fresh profile. Installed uBO 1.13.8 and the page from pcmag.com loaded correctly as I reported on my previous post. 
Let me note that I haven't changed any filters...
So can you please explain your :  
> It's a filter issue, fixed in uBO filters.

BTW - I can reproduce the issue in 1.13.8 ONLY IF I install uBO 1.14.14 before.

**Comment by gorhill** [comment]
> So can you please explain your

Because you failed to force an update of the filter lists right after installing 1.13.8 and before testing the filter issue on `pcmag.com`.

**Comment by LL25255252** [comment]
OK. I forced a filter update and now no more issues on pcmag (tested on all versions).

[suggestion] - Let the addon itself run that filter update right after the addon is being installed.

I checked the addon's homepage (in AMO and here at github) and as far as i checked it doesn't remind the user to open the settings page and force filter update after installation.

BTW is there somewhere a documentation of you adjusting that filter ?
My "automatic filters update" checkbox is set to ON but maybe I'll find it helpful. 

Thanks again in advance !

**Comment by LL25255252** [comment]
OK. I forced a filter update and now no more issues on pcmag (tested on all versions).

[suggestion] - Let the addon itself run that filter update right after it is being installed.

I checked the addon's homepage (in AMO and here at github) and as far as i checked it doesn't remind the user to open the settings page and force filter update after installation.

BTW is there somewhere a documentation of you adjusting that filter ?
My "automatic filters update" checkbox is set to ON but maybe I'll find it helpful. 

Thanks again in advance !
